### DISCUSSION TITLE
Heat conduction through a hollow steel section

### DISCUSSION THREAD

**Discussion by R-Godakandage** [firemodels/fds#12992] [Q&A] [answered]
Hi, 
I have needed to model heat conduction through a square hollow section (steel). So I did a simple model to identify whether this is possible. I think I should use HT3D for this simulation. But it didn't worked. (I used inert filing within the hollow part to avoid heat transfer through radiation and convection) I highly appreciate your views and suggestions on this regard. Thanks in advance.

![image](https://github.com/firemodels/fds/assets/170778336/8ab59a46-55f3-4e91-9ddc-be0abe0eaff3)
![image](https://github.com/firemodels/fds/assets/170778336/00825c6d-4115-42c1-81c0-c59602eba20f)
![image](https://github.com/firemodels/fds/assets/170778336/abc89259-f9e7-4b67-b897-0ed4e496c656)

**Comment by mcgratta** [comment, marked as answer]
We do not know how you set up the case. Post a very simple input file for the case above. Also, make sure you are using the latest version of the code.

**Reply by R-Godakandage** [reply]
Dr. Kevin I tried with fds 6.8.0, and now it is working. Thanks a lot for your swift reply.

No,  I'm trying to model a 3m height steel column having a thickness of 1.2mm . As I understoo , HT3D uses the mesh size rather than the material thickness. In my case, using 1.2mm fine mesh seems to be computationally expensive.  So do you have any suggestion to work around this without going for a finer mesh? 

Thanks in advance.

**Reply by mcgratta** [reply]
First, use the latest version, 6.9.1. The HT3D feature has been evolving. You can make a column out of thin plates and the heat transfer will take place in the lateral direction. The heat transfer in the normal direction will not rely on the mesh thickness.

**Reply by R-Godakandage** [reply]
Thanks a lot Dr. Kevin. I'll check the latest version.

**Reply by R-Godakandage** [reply]
Dr. Kevin,

I tried a simple case of thin plate column with HT3D, and tried to run in a HPC (48 vcpus ). How ever with HT3D enabled it doesnt run in the HPC giving segmentation error. However, without HT3D it runs. Im using fds 6.9.1, Im adding the error file and my fds file could you please let me know whether there's a error in my fds code?

test123.fds
Generated by PyroSim 2023.3.1312
May 30, 2024, 2:59:58 PM

&HEAD CHID='test123'/
&TIME T_END=100.0/
&DUMP DT_RESTART=300.0, DT_SL3D=0.25/

&MESH ID='Mesh01-01', IJK=4,4,20, XB=0.0,0.4,0.0,0.4,0.0,2.0/
&MESH ID='Mesh01-02', IJK=4,4,20, XB=0.0,0.4,0.4,0.8,0.0,2.0/
&MESH ID='Mesh01-03', IJK=4,4,20, XB=0.0,0.4,0.8,1.2,0.0,2.0/
&MESH ID='Mesh01-04', IJK=4,4,20, XB=0.0,0.4,1.2,1.6,0.0,2.0/
&MESH ID='Mesh01-05', IJK=4,4,20, XB=0.0,0.4,1.6,2.0,0.0,2.0/
&MESH ID='Mesh01-06', IJK=4,4,20, XB=0.4,0.8,0.0,0.4,0.0,2.0/
&MESH ID='Mesh01-07', IJK=4,4,20, XB=0.4,0.8,0.4,0.8,0.0,2.0/
&MESH ID='Mesh01-08', IJK=4,4,20, XB=0.4,0.8,0.8,1.2,0.0,2.0/
&MESH ID='Mesh01-09', IJK=4,4,20, XB=0.4,0.8,1.2,1.6,0.0,2.0/
&MESH ID='Mesh01-10', IJK=4,4,20, XB=0.4,0.8,1.6,2.0,0.0,2.0/
&MESH ID='Mesh01-11', IJK=4,4,20, XB=0.8,1.2,0.0,0.4,0.0,2.0/
&MESH ID='Mesh01-12', IJK=4,4,20, XB=0.8,1.2,0.4,0.8,0.0,2.0/
&MESH ID='Mesh01-13', IJK=4,4,20, XB=0.8,1.2,0.8,1.2,0.0,2.0/
&MESH ID='Mesh01-14', IJK=4,4,20, XB=0.8,1.2,1.2,1.6,0.0,2.0/
&MESH ID='Mesh01-15', IJK=4,4,20, XB=0.8,1.2,1.6,2.0,0.0,2.0/
&MESH ID='Mesh01-16', IJK=4,4,20, XB=1.2,1.6,0.0,0.4,0.0,2.0/
&MESH ID='Mesh01-17', IJK=4,4,20, XB=1.2,1.6,0.4,0.8,0.0,2.0/
&MESH ID='Mesh01-18', IJK=4,4,20, XB=1.2,1.6,0.8,1.2,0.0,2.0/
&MESH ID='Mesh01-19', IJK=4,4,20, XB=1.2,1.6,1.2,1.6,0.0,2.0/
&MESH ID='Mesh01-20', IJK=4,4,20, XB=1.2,1.6,1.6,2.0,0.0,2.0/
&MESH ID='Mesh01-21', IJK=4,4,20, XB=1.6,2.0,0.0,0.4,0.0,2.0/
&MESH ID='Mesh01-22', IJK=4,4,20, XB=1.6,2.0,0.4,0.8,0.0,2.0/
&MESH ID='Mesh01-23', IJK=4,4,20, XB=1.6,2.0,0.8,1.2,0.0,2.0/
&MESH ID='Mesh01-24', IJK=4,4,20, XB=1.6,2.0,1.2,1.6,0.0,2.0/
&MESH ID='Mesh01-25', IJK=4,4,20, XB=1.6,2.0,1.6,2.0,0.0,2.0/

&SPEC ID='REAC_FUEL', FORMULA='C2H6O1'/

&REAC ID='METHANE',
      FYI='AFT NIST Multi-Floor FDS5 Validation',
      FUEL='REAC_FUEL',
      RADIATIVE_FRACTION=0.35/

&MATL ID='STEEL',
      FYI='Drysdale, Intro to Fire Dynamics - ATF NIST Multi-Floor Validation',
      SPECIFIC_HEAT=0.46,
      CONDUCTIVITY=45.8,
      DENSITY=7850.0,
      EMISSIVITY=0.95/

&SURF ID='Surface01',
      COLOR='RED',
      HRRPUA=1000.0,
      TMP_FRONT=300.0/
&SURF ID='steel',
      RGB=146,202,166,
      MATL_ID(1,1)='STEEL',
      MATL_MASS_FRACTION(1,1)=1.0,
      HT3D=.TRUE.,
      CELL_SIZE=0.1/

&OBST ID='Obstruction', XB=0.8,1.3,0.7,1.2,0.0,0.1, SURF_IDS='Surface01','INERT','INERT'/ 
&OBST ID='Obstruction', XB=0.7,0.7,0.7,1.2,0.1,1.9, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=0.2,0.7,0.7,0.7,0.1,1.9, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=0.2,0.7,1.2,1.2,0.1,1.9, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=0.2,0.2,0.7,1.2,0.1,1.9, SURF_ID='steel'/ 

&VENT ID='Mesh Vent: Mesh01-01 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.0,0.4,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-01 [YMIN]', SURF_ID='OPEN', XB=0.0,0.4,0.0,0.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-01 [ZMAX]', SURF_ID='OPEN', XB=0.0,0.4,0.0,0.4,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-01 [ZMIN]', SURF_ID='OPEN', XB=0.0,0.4,0.0,0.4,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-02 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.4,0.8,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-02 [ZMAX]', SURF_ID='OPEN', XB=0.0,0.4,0.4,0.8,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-02 [ZMIN]', SURF_ID='OPEN', XB=0.0,0.4,0.4,0.8,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-03 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.8,1.2,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-03 [ZMAX]', SURF_ID='OPEN', XB=0.0,0.4,0.8,1.2,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-03 [ZMIN]', SURF_ID='OPEN', XB=0.0,0.4,0.8,1.2,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-04 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,1.2,1.6,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-04 [ZMAX]', SURF_ID='OPEN', XB=0.0,0.4,1.2,1.6,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-04 [ZMIN]', SURF_ID='OPEN', XB=0.0,0.4,1.2,1.6,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-05 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,1.6,2.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-05 [YMAX]', SURF_ID='OPEN', XB=0.0,0.4,2.0,2.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-05 [ZMAX]', SURF_ID='OPEN', XB=0.0,0.4,1.6,2.0,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-05 [ZMIN]', SURF_ID='OPEN', XB=0.0,0.4,1.6,2.0,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-06 [YMIN]', SURF_ID='OPEN', XB=0.4,0.8,0.0,0.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-06 [ZMAX]', SURF_ID='OPEN', XB=0.4,0.8,0.0,0.4,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-06 [ZMIN]', SURF_ID='OPEN', XB=0.4,0.8,0.0,0.4,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-07 [ZMAX]', SURF_ID='OPEN', XB=0.4,0.8,0.4,0.8,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-07 [ZMIN]', SURF_ID='OPEN', XB=0.4,0.8,0.4,0.8,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-08 [ZMAX]', SURF_ID='OPEN', XB=0.4,0.8,0.8,1.2,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-08 [ZMIN]', SURF_ID='OPEN', XB=0.4,0.8,0.8,1.2,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-09 [ZMAX]', SURF_ID='OPEN', XB=0.4,0.8,1.2,1.6,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-09 [ZMIN]', SURF_ID='OPEN', XB=0.4,0.8,1.2,1.6,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-10 [YMAX]', SURF_ID='OPEN', XB=0.4,0.8,2.0,2.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-10 [ZMAX]', SURF_ID='OPEN', XB=0.4,0.8,1.6,2.0,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-10 [ZMIN]', SURF_ID='OPEN', XB=0.4,0.8,1.6,2.0,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-11 [YMIN]', SURF_ID='OPEN', XB=0.8,1.2,0.0,0.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-11 [ZMAX]', SURF_ID='OPEN', XB=0.8,1.2,0.0,0.4,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-11 [ZMIN]', SURF_ID='OPEN', XB=0.8,1.2,0.0,0.4,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-12 [ZMAX]', SURF_ID='OPEN', XB=0.8,1.2,0.4,0.8,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-12 [ZMIN]', SURF_ID='OPEN', XB=0.8,1.2,0.4,0.8,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-13 [ZMAX]', SURF_ID='OPEN', XB=0.8,1.2,0.8,1.2,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-13 [ZMIN]', SURF_ID='OPEN', XB=0.8,1.2,0.8,1.2,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-14 [ZMAX]', SURF_ID='OPEN', XB=0.8,1.2,1.2,1.6,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-14 [ZMIN]', SURF_ID='OPEN', XB=0.8,1.2,1.2,1.6,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-15 [YMAX]', SURF_ID='OPEN', XB=0.8,1.2,2.0,2.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-15 [ZMAX]', SURF_ID='OPEN', XB=0.8,1.2,1.6,2.0,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-15 [ZMIN]', SURF_ID='OPEN', XB=0.8,1.2,1.6,2.0,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-16 [YMIN]', SURF_ID='OPEN', XB=1.2,1.6,0.0,0.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-16 [ZMAX]', SURF_ID='OPEN', XB=1.2,1.6,0.0,0.4,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-16 [ZMIN]', SURF_ID='OPEN', XB=1.2,1.6,0.0,0.4,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-17 [ZMAX]', SURF_ID='OPEN', XB=1.2,1.6,0.4,0.8,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-17 [ZMIN]', SURF_ID='OPEN', XB=1.2,1.6,0.4,0.8,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-18 [ZMAX]', SURF_ID='OPEN', XB=1.2,1.6,0.8,1.2,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-18 [ZMIN]', SURF_ID='OPEN', XB=1.2,1.6,0.8,1.2,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-19 [ZMAX]', SURF_ID='OPEN', XB=1.2,1.6,1.2,1.6,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-19 [ZMIN]', SURF_ID='OPEN', XB=1.2,1.6,1.2,1.6,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-20 [YMAX]', SURF_ID='OPEN', XB=1.2,1.6,2.0,2.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-20 [ZMAX]', SURF_ID='OPEN', XB=1.2,1.6,1.6,2.0,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-20 [ZMIN]', SURF_ID='OPEN', XB=1.2,1.6,1.6,2.0,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-21 [XMAX]', SURF_ID='OPEN', XB=2.0,2.0,0.0,0.4,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-21 [YMIN]', SURF_ID='OPEN', XB=1.6,2.0,0.0,0.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-21 [ZMAX]', SURF_ID='OPEN', XB=1.6,2.0,0.0,0.4,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-21 [ZMIN]', SURF_ID='OPEN', XB=1.6,2.0,0.0,0.4,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-22 [XMAX]', SURF_ID='OPEN', XB=2.0,2.0,0.4,0.8,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-22 [ZMAX]', SURF_ID='OPEN', XB=1.6,2.0,0.4,0.8,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-22 [ZMIN]', SURF_ID='OPEN', XB=1.6,2.0,0.4,0.8,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-23 [XMAX]', SURF_ID='OPEN', XB=2.0,2.0,0.8,1.2,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-23 [ZMAX]', SURF_ID='OPEN', XB=1.6,2.0,0.8,1.2,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-23 [ZMIN]', SURF_ID='OPEN', XB=1.6,2.0,0.8,1.2,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-24 [XMAX]', SURF_ID='OPEN', XB=2.0,2.0,1.2,1.6,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-24 [ZMAX]', SURF_ID='OPEN', XB=1.6,2.0,1.2,1.6,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-24 [ZMIN]', SURF_ID='OPEN', XB=1.6,2.0,1.2,1.6,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh01-25 [XMAX]', SURF_ID='OPEN', XB=2.0,2.0,1.6,2.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-25 [YMAX]', SURF_ID='OPEN', XB=1.6,2.0,2.0,2.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-25 [ZMAX]', SURF_ID='OPEN', XB=1.6,2.0,1.6,2.0,2.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01-25 [ZMIN]', SURF_ID='OPEN', XB=1.6,2.0,1.6,2.0,0.0,0.0/ 

&BNDF QUANTITY='WALL TEMPERATURE'/


&TAIL /


**Error message** 

Fire Dynamics Simulator

 Current Date     : May 30, 2024  04:56:19
 Revision         : FDS-6.9.1-0-g889da6a-release
 Revision Date    : Sun Apr 7 17:05:06 2024 -0400
 Compiler         : Intel(R) Fortran Intel(R) 64 Compiler Classic for applications running on Intel(R) 64, Version 2021.7.1 Build 20221019_000000
 Compilation Date : Apr 09, 2024 13:24:51

 Number of MPI Processes:  25

 MPI version: 3.1
 MPI library version: Intel(R) MPI Library 2021.6 for Linux* OS


 Job TITLE        :
 Job ID string    : test123

 Time Step:       1, Simulation Time:      0.11 s
forrtl: severe (174): SIGSEGV, segmentation fault occurred
Image              PC                Routine            Line        Source
libpthread-2.31.s  00001478EF607420  Unknown               Unknown  Unknown
fds                0000000006B32BF5  Unknown               Unknown  Unknown
fds                0000000006B25796  Unknown               Unknown  Unknown
fds                0000000006B121A0  Unknown               Unknown  Unknown
fds                0000000006AB8148  Unknown               Unknown  Unknown
fds                000000000040A71D  Unknown               Unknown  Unknown
libc-2.31.so       00001478ED6C8083  __libc_start_main     Unknown  Unknown
fds                000000000040A636  Unknown               Unknown  Unknown
forrtl: severe (174): SIGSEGV, segmentation fault occurred
Image              PC                Routine            Line        Source
libpthread-2.31.s  000015269ADB7420  Unknown               Unknown  Unknown
fds                0000000006B32BF5  Unknown               Unknown  Unknown
fds                0000000006B25796  Unknown               Unknown  Unknown
fds                0000000006B121A0  Unknown               Unknown  Unknown
fds                0000000006AB8148  Unknown               Unknown  Unknown
fds                000000000040A71D  Unknown               Unknown  Unknown
libc-2.31.so       0000152698E78083  __libc_start_main     Unknown  Unknown
fds                000000000040A636  Unknown               Unknown  Unknown
forrtl: severe (174): SIGSEGV, segmentation fault occurred
Image              PC                Routine            Line        Source
libpthread-2.31.s  0000154602D9F420  Unknown               Unknown  Unknown
fds                0000000006B32BF5  Unknown               Unknown  Unknown
fds                0000000006B25796  Unknown               Unknown  Unknown
fds                0000000006B121A0  Unknown               Unknown  Unknown
fds                0000000006AB8148  Unknown               Unknown  Unknown
fds                000000000040A71D  Unknown               Unknown  Unknown
libc-2.31.so       0000154600E60083  __libc_start_main     Unknown  Unknown
fds                000000000040A636  Unknown               Unknown  Unknown
forrtl: severe (174): SIGSEGV, segmentation fault occurred
Image              PC                Routine            Line        Source
libpthread-2.31.s  0000151403120420  Unknown               Unknown  Unknown
fds                0000000006B32BF5  Unknown               Unknown  Unknown
fds                0000000006B25796  Unknown               Unknown  Unknown
fds                0000000006B121A0  Unknown               Unknown  Unknown
fds                0000000006AB8148  Unknown               Unknown  Unknown
fds                000000000040A71D  Unknown               Unknown  Unknown
libc-2.31.so       00001514011E1083  __libc_start_main     Unknown  Unknown
fds                000000000040A636  Unknown               Unknown  Unknown
Abort(255982479) on node 11 (rank 11 in comm 0): Fatal error in PMPI_Testall: Other MPI error, error stack:
PMPI_Testall(362)..............: MPI_Testall(count=8, req_array=0x7ffda831bfe0, flag=0x7ffda831c030, status_array=0x1) failed
MPIR_Testall_impl(44)..........:
MPIDI_Progress_test(95)........:
MPIDI_OFI_handle_cq_error(1100): OFI poll failed (ofi_events.c:1100:MPIDI_OFI_handle_cq_error:Input/output error)
Abort(390200207) on node 12 (rank 12 in comm 0): Fatal error in PMPI_Testall: Other MPI error, error stack:
PMPI_Testall(362)..............: MPI_Testall(count=8, req_array=0x7ffc4272f0a0, flag=0x7ffc4272f0f0, status_array=0x1) failed
MPIR_Testall_impl(44)..........:
MPIDI_Progress_test(95)........:
MPIDI_OFI_handle_cq_error(1100): OFI poll failed (ofi_events.c:1100:MPIDI_OFI_handle_cq_error:Input/output error)




![image](https://github.com/firemodels/fds/assets/170778336/58ca018a-2d9c-41cc-b4b5-17040506ca23)

**Comment by mcgratta** [comment]
I will take a look at this.

**Comment by mcgratta** [comment]
There should be an `ERROR` statement that would have stopped the case immediately. The way thin obstructions work with `HT3D` is that the thickness of the obstruction is conveyed directly through the `XB` bounds on the `OBST` line. In your case, there is no indication in the input file as to the thickness of the steel plates. Read the section in the User's Guide about thin obstructions and `HT3D`. Meanwhile, I will add an `ERROR` statement for this kind of input error.

**Reply by R-Godakandage** [reply]
Thanks a lot Dr. Mcgratten,  it worked . The real issue for the error (segmentation error) seems to be the insufficient memory storage of the hpc. I initially tried with a 32 GB memory and when I used 96 GB memory in my hpc instance it worked with multiple meshes.
